### ISSUE TITLE
get topological navigation to use monitored nav

### ISSUE THREAD

**Issue by bfalacerda** [strands-project/strands_navigation#20]
As we discussed before, the topological navigation should always call the monitored_navigation action server and give it a string saying which continuous nav actionlib server to execute and a "move_base goal" (we assume all continuous nav actionlib server receive the same type of goals as move_base). My latest pull request gets the monitored_navigation to a point where this can be done, we just have to edit the topological_navigation. @Jailander , is the ramp_climb (and others) ready to receive the same type of goals as move_base?

**Comment by Jailander** [comment]
Hi, are sure about this?, 
I mean it is not a big problem, currently ramp_climb needs a timeout only but it can be predefined and a move_base goal can be ignored, and the People friendly navigation from @cdondrup uses move_base goals and the visual navigation server can be adapted to these kind of goals without much trouble.
However, doesn't it make our system less flexible? there can be future ways of navigation that may require more arguments or at least different, for example the lift action server that you wanted to implement. I always thought that the monitored navigation would be only for move base and similar. 
This is not a big problem because servers that are not adaptable can be called directly from the topological navigation, but it would be nice to know your thoughts on this. 
In conclusion, we will adapt the ramp_climb server right now, which is the only one that is being used apart from move_base, and as soon as you tell me how to call the monitored navigation, I will modify the topological navigation to use it.

**Comment by bfalacerda** [comment]
It does decrease a bit of flexibility, but I think it would be good to monitor all the navigations so we can always ask for human help when it fails, and having an arbitrary goal message is harder to do. The goal is of the type http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html , and I think that all navigation goals can be represented like this. If they cannot later we can always add more fields to the message.

Regarding the timeout for the ramp_climb, that sounds more like a parameter than a goal. 

The readme in my new pull request tells you how to run the server. Basically, to write a client for it that is using move_base, you need something like:

```
from strands_navigation_msgs.msg import MonitoredNavigationAction
from strands_navigation_msgs.msg import MonitoredNavigationGoal
...

client= actionlib.SimpleActionClient('monitored_navigation', MonitoredNavigationAction)
goal=MonitoredNavigationGoal()
goal.action_server='move_base'
goal.target_pose.header.frame_id = "map"
goal.target_pose.header.stamp = rospy.get_rostime()
goal.target_pose.pose.position.x = float(inf[0])
goal.target_pose.pose.position.y = float(inf[1])
goal.target_pose.pose.orientation.x = 0
goal.target_pose.pose.orientation.y = 0
goal.target_pose.pose.orientation.z = float(inf[5])
goal.target_pose.pose.orientation.w = float(inf[6])
...
client.send_goal(goal)
...
```

**Comment by hawesie** [comment]
This now appears to be done, reopen if not.
